variables.robot

```diff
@@ -40,9 +40,9 @@ ${OS_UBUNTU}=               ubuntu
 &{RTE10}=                   ip=192.168.10.173    cpuid=02c00042a3b72a65    pcb_rev=0.5.3
 ...                         platform=apu3    board-revision=c4    env=prod
 ...                         platform_vendor=PC Engines    firmware_type=BIOS
-&{RTE11}=                   ip=192.168.10.174    cpuid=02c000426621f7ea    pcb_rev=0.5.3
+&{RTE11}=                   ip=192.168.10.174    cpuid=02c000426621f7ea    pcb_rev=1.0.0
```

Comment:
Are we sure? IIUC most recent version is v1.1.0

Reply:
That's what snipeit contains, I haven't checked the actual hardware yet

Reply:
@mkopec I would appreciate if that would be checked, maybe snipeit contain wrong information.

Reply:
> IIUC most recent version is v1.1.0

It is but we do not use it for PC Engines. The v1.1.0 is too precious to use on platforms not strictly requiring it. We are short on RTEs.

Reply:
Also, this information is not used at all in tests.

Reply:
Another thing to remove

Reply:
Removed the pcb_rev variable in commit f6cfcd1883c2c8f023149062acb5f43af98d2daf . It makes more sense to track this type of information in snipeit.